tests/fixtures.py

```diff
@@ -1,51 +1,50 @@
 import pytest
 from tests import helpers
-from ocs_ci.ocs import constants, defaults
-from ocs_ci.utility import templating
+from ocs_ci.ocs import constants
 
 
 @pytest.fixture()
 def create_rbd_secret(request):
     """
-    Create a rbd secret
+    Create an RBD secret
     """
     class_instance = request.node.cls
 
     def finalizer():
         """
-        Delete the project
+        Delete the RBD secret
         """
-        if class_instance.rbd_secret_obj.get():
-            class_instance.rbd_secret_obj.delete()
+        if class_instance.secret_obj.get():
+            class_instance.secret_obj.delete()
 
     request.addfinalizer(finalizer)
 
-    class_instance.rbd_secret_obj = helpers.create_secret(
+    class_instance.secret_obj = helpers.create_secret(
         interface_type=constants.CEPHBLOCKPOOL
     )
-    assert class_instance.rbd_secret_obj, "Failed to create rbd secret"
+    assert class_instance.secret_obj, "Failed to create secret"
 
 
 @pytest.fixture()
 def create_cephfs_secret(request):
     """
-    Create a cephfs secret
+    Create a CephFS secret
     """
     class_instance = request.node.cls
 
     def finalizer():
         """
-        Delete the project
+        Delete the FS secret
         """
-        if class_instance.cephfs_secret_obj.get():
-            class_instance.cephfs_secret_obj.delete()
+        if class_instance.secret_obj.get():
+            class_instance.secret_obj.delete()
 
     request.addfinalizer(finalizer)
 
-    class_instance.cephfs_secret_obj = helpers.create_secret(
+    class_instance.secret_obj = helpers.create_secret(
```

Comment:
Will this work if a test is calling both create_rbd_secret and create_cephfs_secret, as both are using class_instance.secret_obj

Reply:
Good question..I don't think we have tests that create both block pool and cephfs.

Reply:
@sidhant-agrawal anyway, I'll change it to be distinct

Reply:
ok